lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -61,6 +83,8 @@ class AwsCompileScheduledEvents {
                   "ScheduleExpression": "${ScheduleExpression}",
                   "State": "${State}",
                   "Targets": [{
+                    ${Input ? `"Input": "${Input}",` : ``}
+                    ${InputPath ? `"InputPath": "${InputPath}",` : ``}
```

Comment:
ESLint seems to be failing here as well detecting double quotes... 😰 https://travis-ci.org/serverless/serverless/jobs/173694719 

I am not sure but probably it wants the empty string to use normal single quotes... That's annoyng!